src/coral_models/wav2vec2.py

```diff
@@ -179,8 +171,40 @@ def load_trainer_class(self) -> Type[Trainer]:
     def load_compute_metrics(self) -> Callable[[EvalPrediction], dict]:
         return partial(compute_wer_metrics, processor=self.processor)
 
+    def load_training_arguments(self) -> TrainingArguments:
+        args = TrainingArguments(
+            output_dir=self.cfg.model_dir,
+            hub_model_id=self.cfg.hub_id,
+            per_device_train_batch_size=self.cfg.model.batch_size,
+            per_device_eval_batch_size=self.cfg.model.batch_size,
```

Comment:
Curious: Why use same? To avoid wasting time maximizing eval batch size without OOM?

Reply:
I guess yeah. We could probably up the test batch size, but this is just playing it safe